homeassistant/components/cast/media_player.py

```diff
@@ -120,7 +127,6 @@ def _fill_out_missing_chromecast_info(info: ChromecastInfo) -> ChromecastInfo:
 def _discover_chromecast(hass: HomeAssistantType, info: ChromecastInfo):
     if info in hass.data[KNOWN_CHROMECAST_INFO_KEY]:
```

Comment:
Since we now maintain a set of services and let pychromecast worry about resolving IP / host, maybe this set is no longer needed?

Reply:
I think that this might still be useful for people doing a manual configuration of Casts?

Reply:
The manually configured cast still has a UUID though, we don't need two ways to check for uniqueness?

Reply:
Ah, good point ! In that case 🔥

Reply:
OK, I'll leave that for a separate cleanup PR though?

Reply:
Yes